discussion_grading/constants.py

```diff
@@ -0,0 +1,7 @@
+"""This module contains the constants used in the Discussion Grading XBlock."""
```

Comment:
Why do we need a new module for constants if they're used in just one module? Could we consider adding them in the same module? Let me know.

Reply:
Yes, you are right
Done: https://github.com/eduNEXT/xblock-discussion-grading/commit/79799038af7d261558e80dafebb79139f114f433